packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -174,82 +126,76 @@ export function usePriority() {
   };
 }
 
+const mapEncounterProperties = (encounter: Encounter): MappedEncounter => ({
+  diagnoses: encounter.diagnoses,
+  encounterDatetime: encounter.encounterDatetime,
+  encounterType: encounter.encounterType.display,
+  obs: encounter.obs,
+  provider: encounter.encounterProviders[0]?.provider?.person?.display,
+  uuid: encounter.uuid,
+  voided: encounter.voided,
+});
+
+export const mapVisitQueueEntryProperties = (
+  queueEntry: QueueEntry,
+  visitQueueNumberAttributeUuid: string,
+): MappedVisitQueueEntry => ({
+  id: queueEntry.uuid,
+  encounters: queueEntry.visit?.encounters?.map(mapEncounterProperties),
+  name: queueEntry.display,
+  patientUuid: queueEntry.patient.uuid,
+  patientAge: queueEntry.patient.person?.age + '',
+  patientSex: queueEntry.patient.person?.gender === 'M' ? 'MALE' : 'FEMALE',
```

Comment:
Perhaps we should make this translatable

Reply:
This is copy/pasted from old code that I plan to deprecate / delete. I agree that it should be translated, but I think it makes sense to keep as is as it's going away soon.